opm/autodiff/MultisegmentWell_impl.hpp

```diff
@@ -339,8 +339,9 @@ namespace Opm
                     }
                 }
 
-                if (original_rates_under_phase_control != 0.0 ) {
-                    double scaling_factor = target / original_rates_under_phase_control;
+                // sometimes the previous report step is zero-rate control, it causes this value to be some extremely small value
```

Comment:
Two issues:
 - Is original_rates_under_phase_control always positive? I think it can be negative, and that must be taken into account.
 - The 1.0e-10 should be a constant with a name, such as rate_threshold.